packages/manager/modules/adp/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import rollupConfig from '@ovh-ux/component-rollup-config';
+
+const config = rollupConfig({
+  input: 'src/index.js',
+});
+
+const outputs = [config.es()];
+
+if (process.env.BUILD === 'production') {
+  outputs.push(config.cjs());
+  outputs.push(
+    config.umd({
+      output: {
+        globals: {
+          angular: 'angular',
```

Comment:
Is `d3` should be also defined as globals? (I'm not sure about it)

Do you any warning when trying to build the module?

Reply:
d3 was temporarily added to the dependencies to solve an issue in yarn build. It has been removed now, since we actually do not have any dependencies on d3